build.gradle

```diff
@@ -33,10 +35,10 @@ dependencies {
     compile('org.springframework.data:spring-data-elasticsearch') {
         exclude module: 'log4j-api'
     }
-
     compile 'com.google.guava:guava:23.5-jre'
     compile 'org.apache.commons:commons-lang3:3.7'
     compile 'org.codehaus.janino:janino:3.0.8'
+    compile group: 'com.github.vanroy', name: 'spring-boot-starter-data-jest', version: '3.1.1.RELEASE'
```

Comment:
consistency!! Can we keen the deps in the shorter format? e.g. compile 'com.github.vanroy:spring-boot-starter-data-jest:3.1.1.RELEASE'